Xamarin.Forms.Core/IPlatformServices.cs

```diff
@@ -32,5 +32,7 @@ public interface IPlatformServices
 		void StartTimer(TimeSpan interval, Func<bool> callback);
 
 		string RuntimePlatform { get; }
+
+		void QuitApp();
```

Comment:
that'd be the first time (to the best of my knowledge) that we abbreviate Application into App in an API. Go for `QuitApplication` or `ExitApplication` instead.